pywwt/layers.py

```diff
@@ -276,49 +276,48 @@ def add_image_layer(
             Use this parameter to set a display name for the image set.
             If not set, pywwt will set a name based on the file name of the
             provided image.
-        force_hipsgen : optional boolean, defaults to False
-            Force the input FITS to be tiled with HiPSgen. If this and
-            *force_tan* are set to False, this method will figure out when to
-            split the input into tiles and also which type of projection to
-            use. Tangential projection for smaller angular areas and HiPSgen
-            larger regions of the sky.
-        force_tan : optional boolean, defaults to False
-            Force the input FITS to be tiled with a tangential projection. If
-            this and *force_hipsgen* are set to False, this method will figure
-            out when to split the input into tiles and also which type of
-            projection to use. Tangential projection for smaller angular areas
-            and HiPSgen larger regions of the sky.
+        tiling_method : optional :class:`~toasty.TilingMethod`
+            Can be used to force a specific tiling method, i.e. tiled
+            tangential projection, TOAST, HiPS, or even untiled. Defaults
+            to auto-detection, which choses the most appropriate method.
         kwargs
             Additional keyword arguments can be used to set properties on the
             image layer or settings for the `toasty` tiling process. Common
-            toasty settings include ``out_dir``, ``override``, and ``blankval``.
+            toasty settings include ``out_dir``, ``override``, ``blankval``,
+            and ``start``.
             See `toasty.tile_fits`.
 
         Returns
         -------
         layer : :class:`~pywwt.layers.ImageLayer` or a subclass thereof
         """
 
+        if isinstance(image, tuple):
+            image, wcs = image
+            from astropy.io.fits import PrimaryHDU
```

Comment:
For consistency with the code below, this could use `astropy.io.fits.PrimaryHDU` instead of importing the name directly.